install.ps1

```diff
@@ -1,88 +1,66 @@
+#!/usr/bin/env pwsh
 # Copyright 2018 the Deno authors. All rights reserved. MIT license.
 # TODO(everyone): Keep this script simple and easily auditable.
+
 param (
-  [alias("v")]
-  [string] $version
+  [ValidatePattern('^v(\d+).(\d+).(\d+)$')]
+  [String] $Version = 'v0.2.6'
 )
 
-$ErrorActionPreference = "Stop"
-
-# Enable TLS 1.2 since it is required for connections to GitHub.
-[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$ErrorActionPreference = 'Stop'
 
-# Helper functions for pretty terminal output.
-function Write-Part ([string] $Text) {
-  Write-Host $Text -NoNewline
-}
-function Write-Emphasized ([string] $Text) {
-  Write-Host $Text -NoNewLine -ForegroundColor "Yellow"
-}
-function Write-Done {
-  Write-Host " done" -NoNewline -ForegroundColor "Green";
-  Write-Host "."
+if ($PSVersionTable.PSVersion.Major -lt 6) {
+  $IsWindows = $true
+  $IsMacOS = $false
 }
 
-if (-not $version) {
-  # Determine latest Deno release via GitHub API.
-  $latest_release_uri = "https://api.github.com/repos/denoland/deno/releases/latest"
-  Write-Part "Downloading "; Write-Emphasized $latest_release_uri; Write-Part "..."
-  $latest_release_json = Invoke-WebRequest -Uri $latest_release_uri
-  Write-Done
+$DenoDir = if ($IsWindows)
```

Comment:
I'd prefer that you use "normal" formatting for if blocks.

Reply:
Hmm, I tried to make it look like an [iif](https://en.wikipedia.org/wiki/IIf).

How about?

```powershell
if ($IsWindows) {
  $DenoDir = "$Home\.deno\bin"
} else {
  $DenoDir = "$Home/.deno/bin"
}
```